test/parallel/test-async-wrap-uid.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+require('../common');
+const fs = require('fs');
+const assert = require('assert');
+const async_wrap = process.binding('async_wrap');
+
+const storage = new Map();
+async_wrap.setupHooks(init, pre, post, destroy);
+async_wrap.enable();
+
+function init(uid) {
```

Comment:
`test/parallel/test-async-wrap-propagate-parent.js` and `test/parallel/test-async-wrap-disabled-propagate-parent.js` tests that.